app/models/asset.rb

```diff
@@ -70,7 +70,6 @@ class Asset < ApplicationRecord
   scope :include_requests_as_source, -> { includes(:requests_as_source) }
 
   scope :sorted, ->() { order('map_id ASC') }
-  scope :for_summary, -> { includes(:map, :barcodes) }
```

Comment:
Hey, I read your commit message for these scope removals. Can you explain it to me? In what sense is this re-defining a scope? Where is it being defined for the first time? Thanks

Reply:
So in this case, this was actually the first definition of the scope, that then gets re-defined in Receptacle (with exactly the same content).

I decided to remove it from Asset, as its an abstract class, inherited by both Labware and Receptacle, and one that's only really still sticking around because we haven't had time to remove it. In this particular situation, for_summary:

- Is only used for receptacles
- Will only work for receptacles (as labware don't have map)

So it didn't really make sense to maintain the version on asset.